### ISSUE TITLE
Question/Request: Pioneer IR decode from previous project

### ISSUE THREAD

**Issue by irakhlin** [kpishere/homie_heatPump#4]
Hello,
I apologize in advanced about opening an issue under your repo not directly related to the source. I came across a older forum post of yours: https://www.eevblog.com/forum/projects/48-bit-(6-byte)-ir-remote-protocol-and-checksum/ in which you linked to https://github.com/kpishere/GetRawIRData. I am currently working on essentially the same thing and was hoping you may have migrated this into another repo. Please feel free to obviously close this issue.

Thank you very much,

**Comment by kpishere** [comment]
Hmm.  I think you found were it ended up irakhlin.  I've split it apart a little bit in a way to make it more generic.  As of right now, you see two different IR protocols supported, an HAVAC heatpump one and a quite common NEC one.

**Comment by irakhlin** [comment]
Hi,

Thank you for responding! For lack of a better place I will leave this here, I have decoded the majority of the message for my mini split (Pioneer mini split). Its composed of two "messages" each 6 bytes each. The second message is easy, its an exact binary invert of the first (flip each one for 0, 0 for 1). From the first message I seem to have worked out the mode, temperature, commands, ect and it seems the checksum is the last 8 bit section but I cant seem to apply the same algorithm you used ( sum of the inverses of each byte minus 1 all inverted). 

I've attached a big list ive been referencing the sequences if you have some time to take a look.
[full_binary_list.txt](https://github.com/kpishere/homie_heatPump/files/5142333/full_binary_list.txt)
